plugins/SitesManager/lang/en.json

```diff
@@ -208,6 +208,9 @@
         "ConsentManagerConnected": "We’ve detected that %1$s is correctly configured to work with Matomo on your site.",
         "SiteWithoutDataReactDescription": "You can use the \"%1$sMatomo Tag Manager%2$s\" to integrate Matomo to track data, follow the instructions from this %3$sguide%4$s.",
         "ReactDetected": "React.js was detected on your website. Did you know you can use the \"%1$sMatomo Tag Manager%2$s\" to integrate Matomo into your site? %3$sLearn more in this guide%4$s.",
-        "ReactDetectedEmail": "React.js was detected on your website. Did you know you can use the \"Matomo Tag Manager\" to track the website data? Learn more here: %1$s"
+        "ReactDetectedEmail": "React.js was detected on your website. Did you know you can use the \"Matomo Tag Manager\" to track the website data? Learn more here: %1$s",
+        "SiteWithoutDataSPA": "SPA / PWA",
+        "SiteWithoutDataSPADescription": "In the rapidly evolving digital landscape, %1$sSingle Page Applications (SPAs)%2$s and %1$sProgressive Web Apps (PWAs)%2$s have emerged as a prevalent choice for developers, given their fast load times, smooth user experience, and offline capabilities. It is easy to start tracking your SPA or PWA using Matomo Analytics. The easiest way to do this is using the Matomo Tag Manager (%3$slearn more%4$s) using the steps below, alternatively you can use the JavaScript Tracking code (%5$sfollow this guide%6$s).",
```

Comment:
>In the rapidly evolving digital landscape, Single Page Applications (SPAs) and Progressive Web Apps (PWAs) have emerged as a prevalent choice for developers, given their fast load times, smooth user experience, and offline capabilities.

Any reason for this marketing blurb for PWAs? We don't have similar glowing oratory for react, vue or even MTM, so it seems a little out of place. If we need to explain what PWAs/SPAs are then I think it would be better to do this in neutral way.

Reply:
@bx80 updated